lib/simplecov/formatter/simple_formatter.rb

```diff
@@ -8,7 +8,7 @@ module Formatter
     class SimpleFormatter
       # Takes a SimpleCov::Result and generates a string out of it
       def format(result)
-        output = "".dup
+        output = +""
```

Comment:
the.... what? I think I need to look this one up...

Reply:
Well... rubocop always teaching me new things. https://www.rubydoc.info/gems/rubocop/0.61.1/RuboCop/Cop/Performance/UnfreezeString